addons/l10n_it_edi/i18n/it.po

```diff
@@ -402,84 +391,64 @@ msgid "INTERNAL"
+#: model:ir.actions.server,name:l10n_it_edi.ir_cron_receive_fattura_pa_invoice_ir_actions_server
+msgid "IT EDI: Receive invoices from the SdI"
+msgstr "IT EDI: Ricezione fatture dal SdI"
+
+#. module: l10n_it_edi
+#. odoo-python
+#: code:addons/l10n_it_edi/models/res_company.py:0
+#, python-format
 msgid ""
-"If PEC Server, only mail from '...@pec.fatturapa.it' will be processed."
-msgstr "Se è un server PEC, solo le email da '...@pec.fatturapa.it' verranno prese in carico."
+"If one of Share Capital or Sole Shareholder is present, then they must be "
+"both filled out."
+msgstr "Se uno fra Capitale Condiviso o Socio Unico è compilato,"
+" allora devono esserlo entrambi."
 
 #. module: l10n_it_edi
+#. odoo-python
 #: code:addons/l10n_it_edi/models/account_invoice.py:0
 #, python-format
 msgid ""
 "If the tax has exoneration, you must enter a kind of exoneration, a law "
 "reference and the amount of the tax must be 0.0."
-msgstr "Se l'Imposta risulta essere un'Esenzione, bisogna compilare il Tipo "
-"di Esenzione e il suo valore dev'essere 0.0."
-
-#. module: l10n_it_edi
-#: model_terms:ir.ui.view,arch_db:l10n_it_edi.res_config_settings_view_form
-msgid ""
-"In demo mode Odoo will just simulate the sending of invoices to the government.<br/>\n"
-"                                        In test mode (experimental) Odoo will send the invoices to a non-production service.\n"
-"                                        Saving this change will direct all companies on this database to this use this configuration.\n"
-"                                        Once registered for testing or official, the mode cannot be changed."
-msgstr "Nella modalitá Demo, Odoo simulerá l'invio delle fatture "
-"all'Agenzia.<br/> Nella modalitá di Test (sperimentale) Odoo manderá le fatture "
-"a un servizio non di produzione. Salvando quest'impostazione fará sì che "
-"tutte le Aziende di questo Database usino questa configurazione. "
-"Una volta registrato per la modalitá Test o Ufficiale, la modalitá non può "
-"più essere cambiata."
-
-#. module: l10n_it_edi
-#: model:ir.model,name:l10n_it_edi.model_fetchmail_server
-msgid "Incoming Mail Server"
-msgstr "Server Email in ingresso"
+msgstr ""
+"Se l'Imposta risulta essere un'Esenzione, bisogna compilare il Tipo di "
+"Esenzione e il suo valore dev'essere 0.0."
 
 #. module: l10n_it_edi
+#. odoo-python
 #: code:addons/l10n_it_edi/models/res_partner.py:0
 #, python-format
 msgid ""
-"Invalid Codice Fiscale '%s': should be like 'MRTMTT91D08F205J' for physical "
-"person and '12345678901' or 'IT12345678901' for businesses."
-msgstr "Codice Fiscale non valido '%s': dev'essere come 'MRTMTT91D08F205J' per "
-"persone fisiche e '12345678901' o 'IT12345678901' per le aziende"
-
-#. module: l10n_it_edi
-#: code:addons/l10n_it_edi/models/account_invoice.py:0
-#, python-format
-msgid "Invalid configuration:"
-msgstr "Configurazione non valida:"
+"Invalid tax code (Codice Fiscale) '%s': should be like 'MRTMTT91D08F205J'"
```

Comment:
Codice Fiscale CANNOT be "IT12345678901". 
The country prefix is always removed from the VAT when it's a business.